src/vtbackend/Functions.h

```diff
@@ -618,10 +623,82 @@ inline auto const& functions() noexcept
     return funcs;
 }
 
+class SupportedSequences
+{
+  public:
+    SupportedSequences()
+    {
+        for (auto [index, seq]: crispy::indexed(functions()))
+        {
+            _supportedSequences[index] = seq;
+        }
+        crispy::sort(
+            _supportedSequences,
+            [](FunctionDefinition const& a, FunctionDefinition const& b) constexpr { return compare(a, b); });
+        _lastIndex = _supportedSequences.size();
```

Comment:
This looks like you're basically copying ALL of the result of `functions()` into the field member `_supportedSequences`. 

The result of `functions()` should be already sorted, then we should rely on that fact.

We could probably ensure that writing `_supportedSequences = functions()` works as well, therefore, you could even further optimize this and get rid of the constructor all along by declaring the field member like this:

```cpp
std::array<FunctionDefinition, functions().size()> _supportedSequences = functions(); // mind, I proposed to rename functions() :)
```